opendbc/car/honda/fingerprints.py

```diff
@@ -926,4 +926,12 @@
       b'28101-65J-N010\x00\x00',
     ],
   },
+  CAR.HONDA_PILOT_4G: {
+    (Ecu.fwdCamera, 0x18dab5f1, None): [
+      b'8S102-T90-A050\x00\x00',
+    ],
+    (Ecu.fwdRadar, 0x18dab0f1, None): [
+      b'8S302-T90-A040\x00\x00',
```

Comment:
Route for these?

Reply:
f9c43864cf057d05|2024-01-15--23-01-20 maybe?

Reply:
Is the tester still around?

Reply:
Unsure, will try to find someone else to get a newer route too